src/index.html

```diff
@@ -17,6 +30,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        />
```

Comment:
The `alt` attribute for the logo image should describe the image content. In this case, it would be better to use a more descriptive text such as 'Moyo company logo'.